packages/mobile/src/app/ErrorScreen.tsx

```diff
@@ -1,9 +1,9 @@
 import FullscreenCTA from '@celo/react-components/components/FullscreenCTA'
-import { componentStyles } from '@celo/react-components/styles/styles'
+import fontFamily from '@celo/react-components/styles/fonts'
```

Comment:
nit: most existing files name it `fontStyles`, we should try to keep the same name everywhere